### DISCUSSION TITLE
Get generated project folder after a build

### DISCUSSION THREAD

**Discussion by Xeladrane91** [codemagic-ci-cd/codemagic-docs#1142] [Q&A] [answered]
Hello,
Is it possible to get the generated project folder after a build ? I see that I can get the runner but I would need the generated source code.
Kind regards

**Comment by himesh-cm** [comment]
@Xeladrane91, what you mean by generated source code? can you please share some reference and elaborate on what you are trying to achieve?

**Comment by Xeladrane91** [comment, marked as answer]
Hello,
I was trying to retrieve the Podfile. I got the answer [here (remote access after build)](https://docs.codemagic.io/troubleshooting/accessing-builder-machine-via-ssh/). 
Have a good day @himesh-cm
